examples/neural_networks/plot_mlp_alpha.py

```diff
@@ -36,7 +36,8 @@
 
 classifiers = []
 for i in alphas:
-    classifiers.append(MLPClassifier(solver='lbfgs', alpha=i, random_state=1,
+    classifiers.append(MLPClassifier(solver='lbfgs', alpha=i,
+                                     random_state=1, max_iter=2000,
```

Comment:
This will likely take a long time.  Could you please time this example before and after this change?

Reply:
It's in the description of the PR. It's actually only 2 sec longer so I think it's fine.

Reply:
Hi @rth , 

Do you mean something different than I did initially ?

From my first message:
With 1000 iterations: python examples/neural_networks/plot_mlp_alpha.py 13,86s user 0,53s system 194% cpu 7,415 total
With 2000 iterations: python examples/neural_networks/plot_mlp_alpha.py 18,73s user 0,60s system 195% cpu 9,873 total

(1000 is before the change, 2000 after)

Reply:
Yes, I missed the timing in the PR description. 

The default is 200 iterations not 1000 though, I think https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html Could you please re-check run time with the default number of iterations?  Are you sure 2000 iterations are necessary. Alternatively maybe we could use max_iter=2000  but with `early_stopping=True`.